framework/include/executioners/Transient.h

```diff
@@ -119,7 +119,7 @@ class Transient : public Executioner
    * Pointer to the TimeStepper
    * @return Pointer to the time stepper for this Executioner
    */
-  TimeStepper * getTimeStepper() { return _time_stepper; }
+  TimeStepper * getTimeStepper() const { return _time_stepper; }
```

Comment:
I would like a `const` implementation that returns a `const TimeStepper *` better

Reply:
Ok, yeah, I can make that change